docs/rules/prefer-stateless-function.md

```diff
@@ -43,3 +44,14 @@ class Foo extends React.Component {
   }
 }
 ```
+
+The following pattern is not considered warnings:
```

Comment:
haha yeah that's what I wrote, but the rest of docs we're like this so I didn't change it. Will fix this.